packages/mask/src/plugins/Avatar/SNSAdaptor/NFTBadge.tsx

```diff
@@ -40,13 +40,14 @@ interface NFTBadgeProps extends withClasses<'root' | 'text' | 'icon'> {
     width?: number
 }
 
-function formatPrice(amount: string) {
+function formatPrice(amount: string, symbol: string, slug: string) {
```

Comment:
No need to pass `slug` and `symbol` into `formatPrice()`. Just handle them in `formatText()`